crates/rpc-client/src/ckb_client.rs

```diff
@@ -97,12 +100,17 @@ impl CKBClient {
     #[instrument(skip_all, fields(tx_hash = %tx_hash.pack()))]
     pub async fn get_transaction(&self, tx_hash: H256) -> Result<Option<Transaction>> {
         let tx_with_status = self.get_transaction_with_status(tx_hash).await?;
-        Ok(tx_with_status
+        tx_with_status
             .and_then(|tx_with_status| tx_with_status.transaction)
             .map(|tv| {
+                let tv: ckb_jsonrpc_types::TransactionView = match tv.inner {
+                    Either::Left(v) => v,
+                    Either::Right(v) => serde_json::from_slice(v.as_bytes())?,
```

Comment:
The bytes are not JSON.

But we won't get a bytes response. We can just use `unreachable!()` here.

Reply:
fixed